src/ATen/native/xpu/mkl/SpectralOps.cpp

```diff
@@ -77,16 +69,15 @@ void _mkl_dft(
   if (!inverse) {
     desc.set_value(config_param::FWD_DISTANCE, idist);
     desc.set_value(config_param::BWD_DISTANCE, odist);
+
+    desc.set_value(config_param::FWD_STRIDES, fwd_strides.data());
+    desc.set_value(config_param::BWD_STRIDES, bwd_strides.data());
   } else {
     desc.set_value(config_param::FWD_DISTANCE, odist);
     desc.set_value(config_param::BWD_DISTANCE, idist);
-  }
 
-  if (!fwd_strides.empty()) {
-    desc.set_value(config_param::FWD_STRIDES, fwd_strides.data());
-  }
-  if (!bwd_strides.empty()) {
-    desc.set_value(config_param::BWD_STRIDES, bwd_strides.data());
+    desc.set_value(config_param::FWD_STRIDES, bwd_strides.data());
+    desc.set_value(config_param::BWD_STRIDES, fwd_strides.data());
```

Comment:
`FWD_STRIDES <- bwd_strides.data()` and `BWD_STRIDES <- fwd_strides.data()` may cause confusion. Please rename `fwd_strides` and `bwd_strides`, or try other methods.